src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-creation-preview-details/osd-creation-preview-details.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+import * as _ from 'lodash';
+
+import { OSDPreview } from '../../../../shared/models/osd-preview.interface';
+
+@Component({
+  selector: 'cd-osd-creation-preview-details',
+  templateUrl: './osd-creation-preview-details.component.html',
+  styleUrls: ['./osd-creation-preview-details.component.scss']
+})
+export class OsdCreationPreviewDetailsComponent implements OnInit {
+  @Input()
+  osd: OSDPreview;
+
+  constructor() {}
+
+  ngOnInit() {}
```

Comment:
You can remove, same to `implements OnInit`.
```suggestion
```

Reply:
Thanks! fixed it.